src/main/java/io/confluent/connect/jdbc/DataConverter.java

```diff
@@ -51,10 +51,17 @@
 
   public static Schema convertSchema(String tableName, ResultSetMetaData metadata)
       throws SQLException {
+    return convertSchema(tableName, null, metadata);
```

Comment:
Ah yes it's left over from a previous version of the code where I was calling `convertSchema` twice, once for the value schema and once for the key schema. I'll remove it.